lib/datadog/appsec/waf/converter.rb

```diff
@@ -8,27 +8,34 @@ module Converter
         module_function
 
         # standard:disable Metrics/MethodLength,Metrics/CyclomaticComplexity
-        def ruby_to_object(val, max_container_size: nil, max_container_depth: nil, max_string_length: nil, coerce: true)
+        def ruby_to_object(val, max_container_size: nil, max_container_depth: nil, max_string_length: nil, top_obj: nil, coerce: true)
           case val
           when Array
             obj = LibDDWAF::Object.new
             res = LibDDWAF.ddwaf_object_array(obj)
             raise ConversionError, "Could not convert into object: #{val}" if res.null?
 
             max_index = max_container_size - 1 if max_container_size
-            unless max_container_depth == 0
+            # TODO: we must pass top object to be able to mark is as truncated
```

Comment:
Is that still needed?

Reply:
removed in [93bc90c](https://github.com/DataDog/libddwaf-rb/pull/88/commits/93bc90c3143656e1953ebb50c068629efd691418)